src/python/pants/bin/remote_pants_runner.py

```diff
@@ -129,9 +273,12 @@ def _run_pants_with_retry(self, pantsd_handle, retries=3):
         attempt += 1
       except NailgunClient.NailgunError as e:
         # Ensure a newline.
-        logger.fatal('')
-        logger.fatal('lost active connection to pantsd!')
-        raise_with_traceback(self._extract_remote_exception(pantsd_handle.pid, e))
+        error_log_msg = '\nlost active connection to pantsd!'
+        ExceptionSink.log_exception(error_log_msg)
```

Comment:
this wont be required at all, since you are already handling this earlier

Reply:
This file has been reverted entirely!